tasks/build-website-development.js

```diff
@@ -0,0 +1,8 @@
+var gulp = require('gulp');
+var config = require('./config.js');
+
+gulp.task('build-website-development', ['prepare-website'], buildWebsiteEevelopmentTask);
+
+function buildWebsiteEevelopmentTask (cb) {
```

Comment:
I dont understand, you talk about middlewares (array of tasks), or named the callback task (the function in last argument)?